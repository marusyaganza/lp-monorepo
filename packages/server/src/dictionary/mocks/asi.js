const asi = [
  {
    meta: {
      id: 'asi:1',
      uuid: 'c68c2865-eb3e-4e93-9ca1-18d9029850f2',
      lang: 'es',
      sort: '2602157000',
      src: 'spanish',
      section: 'alpha',
      stems: [
        'así',
        'así así',
        'así como',
        'así como así',
        'así nomás',
        'así de',
        'así mismo',
        'así que',
        'así y todo'
      ],
      offensive: false
    },
    hom: 1,
    hwi: {
      hw: 'así',
      prs: [
        {
          sound: {
            audio: 'asi0001sp'
          }
        }
      ]
    },
    fl: 'adverb',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                sn: '1',
                dt: [
                  [
                    'text',
                    '{bc}{a_link|like this}, {a_link|like that}, {a_link|so} '
                  ],
                  [
                    'vis',
                    [
                      {
                        t: 'así se hace',
                        tr: "that's how it's done"
                      },
                      {
                        t: 'no puede seguir así',
                        tr: "it can't go on like this"
                      },
                      {
                        t: 'así sea',
                        tr: 'so be it'
                      },
                      {
                        t: 'y así sucesivamente',
                        tr: 'and so on'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '2',
                vrs: [
                  {
                    va: 'así así'
                  }
                ],
                dt: [['text', '{bc}{a_link|so-so}, {a_link|fair}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '3',
                vrs: [
                  {
                    va: 'así como',
                    vac: '~ como'
                  }
                ],
                dt: [['text', '{bc}{a_link|as well as}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '4',
                vrs: [
                  {
                    va: 'así como así'
                  },
                  {
                    vl: 'or',
                    va: 'así nomás'
                  }
                ],
                dt: [['text', '{bc}just like that']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '5',
                vrs: [
                  {
                    va: 'así de',
                    vac: '~ de'
                  }
                ],
                dt: [
                  ['text', '{bc}{a_link|so}, about so '],
                  [
                    'vis',
                    [
                      {
                        t: 'una caja así de grande',
                        tr: 'a box about so big'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sen',
              {
                sn: '6',
                vrs: [
                  {
                    va: 'así mismo'
                  }
                ],
                xrs: [
                  [
                    {
                      xrt: 'asimismo',
                      xref: 'asimismo'
                    }
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '7',
                vrs: [
                  {
                    va: 'así que'
                  }
                ],
                dt: [['text', '{bc}{a_link|so}, {a_link|therefore}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '8',
                vrs: [
                  {
                    va: 'así y todo'
                  }
                ],
                dt: [['text', '{bc}{a_link|even so}']]
              }
            ]
          ]
        ]
      }
    ],
    shortdef: ['like this, like that, so', 'so-so, fair', 'as well as']
  },
  {
    meta: {
      id: 'asi:2',
      uuid: 'ab2d1ae3-4393-496d-bff9-e4bb5a4249f1',
      lang: 'es',
      sort: '2602158000',
      src: 'spanish',
      section: 'alpha',
      stems: ['así'],
      offensive: false
    },
    hom: 2,
    hwi: {
      hw: 'así',
      prs: [
        {
          sound: {
            audio: 'asi0002sp'
          }
        }
      ]
    },
    fl: 'adjective',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                dt: [
                  ['text', '{bc}{a_link|such}, such a, {a_link|like that} '],
                  [
                    'vis',
                    [
                      {
                        t: 'un talento así',
                        tr: 'a talent like that'
                      },
                      {
                        t: 'algo así',
                        tr: 'something like that'
                      },
                      {
                        t: 'así es la vida',
                        tr: "that's life"
                      },
                      {
                        t: 'si es así …',
                        tr: "if that's the case …"
                      }
                    ]
                  ]
                ]
              }
            ]
          ]
        ]
      }
    ],
    shortdef: ['such, such a, like that']
  },
  {
    meta: {
      id: 'asi:3',
      uuid: 'efad688a-6580-4ca1-b144-54f184e9342a',
      lang: 'es',
      sort: '2602159000',
      src: 'spanish',
      section: 'alpha',
      stems: ['así'],
      offensive: false
    },
    hom: 3,
    hwi: {
      hw: 'así',
      prs: [
        {
          sound: {
            audio: 'asi0003sp'
          }
        }
      ]
    },
    fl: 'conjunction',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                dt: [
                  [
                    'text',
                    '{sx|aunque||} {bc}{a_link|even if}, {a_link|even though} '
                  ],
                  [
                    'vis',
                    [
                      {
                        t: 'no irá, así le paguen',
                        tr: "he won't go, even if they pay him"
                      }
                    ]
                  ]
                ]
              }
            ]
          ]
        ]
      }
    ],
    shortdef: ['aunque : even if, even though']
  },
  {
    meta: {
      id: 'aun',
      uuid: 'b5d076af-9a0d-4f1f-b6e6-b4628b712524',
      lang: 'es',
      sort: '2602406000',
      src: 'spanish',
      section: 'alpha',
      stems: ['aun', 'aun así', 'aun más'],
      offensive: false
    },
    hwi: {
      hw: 'aun',
      prs: [
        {
          sound: {
            audio: 'aun0001sp'
          }
        }
      ]
    },
    fl: 'adverb',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                sn: '1',
                dt: [
                  ['text', '{bc}{a_link|even} '],
                  [
                    'vis',
                    [
                      {
                        t: 'ni aun en coche llegaría a tiempo',
                        tr: "I wouldn't arrive on time even if I drove"
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '2',
                vrs: [
                  {
                    va: 'aun así'
                  }
                ],
                dt: [['text', '{bc}{a_link|even so}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '3',
                vrs: [
                  {
                    va: 'aun más'
                  }
                ],
                dt: [['text', '{bc}even more']]
              }
            ]
          ]
        ]
      }
    ],
    shortdef: ['even', 'even so', 'even more']
  },
  {
    meta: {
      id: 'como:3',
      uuid: '580e2bcb-e643-44fa-8d11-81cb5db5ceb4',
      lang: 'es',
      sort: '2605203000',
      src: 'spanish',
      section: 'alpha',
      stems: ['como', 'así como'],
      offensive: false
    },
    hom: 3,
    hwi: {
      hw: 'como',
      prs: [
        {
          sound: {
            audio: 'como004sp'
          }
        }
      ]
    },
    fl: 'preposition',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                sn: '1',
                dt: [
                  ['text', '{bc}{a_link|like}, {a_link|as} '],
                  [
                    'vis',
                    [
                      {
                        t: 'ligero como una pluma',
                        tr: 'light as a feather'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '2',
                vrs: [
                  {
                    va: 'así como'
                  }
                ],
                dt: [['text', '{bc}{a_link|as well as}']]
              }
            ]
          ]
        ]
      }
    ],
    shortdef: ['like, as', 'as well as']
  },
  {
    meta: {
      id: 'decir:1',
      uuid: '48a4e281-4007-423c-9bb9-05dbae10f766',
      lang: 'es',
      sort: '2606512000',
      src: 'spanish',
      section: 'alpha',
      stems: [
        'decir',
        'como quien dice',
        'es decir',
        'dicho y hecho',
        '(o) mejor dicho',
        '¡no me digas!',
        'por así decirlo',
        'querer decir',
        'decirse'
      ],
      offensive: false
    },
    hom: 1,
    hwi: {
      hw: 'decir',
      prs: [
        {
          sound: {
            audio: 'decir01sp'
          }
        }
      ]
    },
    fl: 'transitive verb',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                sn: '1',
                dt: [
                  ['text', '{bc}to {a_link|say} '],
                  [
                    'vis',
                    [
                      {
                        t: 'dice que no irá',
                        tr: "she says she won't go"
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '2',
                dt: [
                  ['text', '{bc}to {a_link|tell} '],
                  [
                    'vis',
                    [
                      {
                        t: 'dime lo que estás pensando',
                        tr: "tell me what you're thinking"
                      },
                      {
                        t: 'ya te lo decía yo',
                        tr: 'I told you so'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '3',
                dt: [
                  ['text', '{bc}to {a_link|tell}, to {a_link|say} '],
                  [
                    'vis',
                    [
                      {
                        t: 'haz lo que te digo',
                        tr: 'do as I say, do what I tell you'
                      },
                      {
                        t: 'te dije que callaras',
                        tr: 'I told you to be quiet'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '4',
                dt: [
                  ['text', '{bc}to {a_link|speak}, to {a_link|talk} '],
                  [
                    'vis',
                    [
                      {
                        t: 'no digas tonterías',
                        tr: "don't talk nonsense"
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '5',
                dt: [
                  ['text', '{bc}to {a_link|call} '],
                  [
                    'vis',
                    [
                      {
                        t: 'me dicen Rosy',
                        tr: 'they call me Rosy'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '6',
                vrs: [
                  {
                    va: 'como quien dice'
                  }
                ],
                dt: [['text', '{bc}so to speak']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '7',
                vrs: [
                  {
                    va: 'es decir'
                  }
                ],
                dt: [['text', '{bc}{a_link|that is to say}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '8',
                vrs: [
                  {
                    va: 'dicho y hecho'
                  }
                ],
                dt: [['text', '{bc}{a_link|no sooner said than done}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '9',
                vrs: [
                  {
                    va: '(o) mejor dicho'
                  }
                ],
                dt: [['text', '{bc}(or) {a_link|rather}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '10',
                vrs: [
                  {
                    va: '¡no me digas!'
                  }
                ],
                dt: [['text', "{bc}you're kidding!, {a_link|you don't say!}"]]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '11',
                vrs: [
                  {
                    va: 'por así decirlo'
                  }
                ],
                dt: [['text', '{bc}so to speak']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '12',
                vrs: [
                  {
                    va: 'querer decir'
                  }
                ],
                dt: [
                  ['text', '{bc}to {a_link|mean} '],
                  [
                    'vis',
                    [
                      {
                        t: '¿qué quiere decir?',
                        tr: 'what do you mean?'
                      }
                    ]
                  ]
                ]
              }
            ]
          ]
        ]
      }
    ],
    dros: [
      {
        drp: 'decirse',
        fl: 'reflexive verb',
        def: [
          {
            sseq: [
              [
                [
                  'sense',
                  {
                    sn: '1',
                    dt: [['text', '{bc}to say to oneself']]
                  }
                ]
              ],
              [
                [
                  'sense',
                  {
                    sn: '2',
                    dt: [
                      ['text', '{bc}to be said '],
                      [
                        'vis',
                        [
                          {
                            t: '¿cómo se dice {ldquo}lápiz{rdquo} en francés?',
                            tr: 'how do you say {ldquo}pencil{rdquo} in French?'
                          }
                        ]
                      ]
                    ]
                  }
                ]
              ]
            ]
          }
        ]
      }
    ],
    suppl: {
      cjts: [
        {
          cjid: 'gppt',
          cjfs: ['diciendo', 'dicho']
        },
        {
          cjid: 'pind',
          cjfs: ['digo', 'dices', 'dice', 'decimos', 'decís', 'dicen']
        },
        {
          cjid: 'pret',
          cjfs: ['decía', 'decías', 'decía', 'decíamos', 'decíais', 'decían']
        },
        {
          cjid: 'pprf',
          cjfs: ['dije', 'dijiste', 'dijo', 'dijimos', 'dijisteis', 'dijeron']
        },
        {
          cjid: 'futr',
          cjfs: ['diré', 'dirás', 'dirá', 'diremos', 'diréis', 'dirán']
        },
        {
          cjid: 'cond',
          cjfs: ['diría', 'dirías', 'diría', 'diríamos', 'diríais', 'dirían']
        },
        {
          cjid: 'psub',
          cjfs: ['diga', 'digas', 'diga', 'digamos', 'digáis', 'digan']
        },
        {
          cjid: 'pisb1',
          cjfs: [
            'dijera',
            'dijeras',
            'dijera',
            'dijéramos',
            'dijerais',
            'dijeran'
          ]
        },
        {
          cjid: 'pisb2',
          cjfs: [
            'dijese',
            'dijeses',
            'dijese',
            'dijésemos',
            'dijeseis',
            'dijesen'
          ]
        },
        {
          cjid: 'fsub',
          cjfs: [
            'dijere',
            'dijeres',
            'dijere',
            'dijéremos',
            'dijereis',
            'dijeren'
          ]
        },
        {
          cjid: 'ppci',
          cjfs: [
            'he dicho',
            'has dicho',
            'ha dicho',
            'hemos dicho',
            'habéis dicho',
            'han dicho'
          ]
        },
        {
          cjid: 'ppsi',
          cjfs: [
            'había dicho',
            'habías dicho',
            'había dicho',
            'habíamos dicho',
            'habíais dicho',
            'habían dicho'
          ]
        },
        {
          cjid: 'pant',
          cjfs: [
            'hube dicho',
            'hubiste dicho',
            'hubo dicho',
            'hubimos dicho',
            'hubisteis dicho',
            'hubieron dicho'
          ]
        },
        {
          cjid: 'fpin',
          cjfs: [
            'habré dicho',
            'habrás dicho',
            'habrá dicho',
            'habremos dicho',
            'habréis dicho',
            'habrán dicho'
          ]
        },
        {
          cjid: 'cpef',
          cjfs: [
            'habría dicho',
            'habrías dicho',
            'habría dicho',
            'habríamos dicho',
            'habríais dicho',
            'habrían dicho'
          ]
        },
        {
          cjid: 'ppfs',
          cjfs: [
            'haya dicho',
            'hayas dicho',
            'haya dicho',
            'hayamos dicho',
            'hayáis dicho',
            'hayan dicho'
          ]
        },
        {
          cjid: 'ppss1',
          cjfs: [
            'hubiera dicho',
            'hubieras dicho',
            'hubiera dicho',
            'hubiéramos dicho',
            'hubierais dicho',
            'hubieran dicho'
          ]
        },
        {
          cjid: 'ppss2',
          cjfs: [
            'hubiese dicho',
            'hubieses dicho',
            'hubiese dicho',
            'hubiésemos dicho',
            'hubieseis dicho',
            'hubiesen dicho'
          ]
        },
        {
          cjid: 'fpsb',
          cjfs: [
            'hubiere dicho',
            'hubieres dicho',
            'hubiere dicho',
            'hubiéremos dicho',
            'hubiereis dicho',
            'hubieren dicho'
          ]
        },
        {
          cjid: 'impf',
          cjfs: ['-', 'di', 'diga', 'digamos', 'decid', 'digan']
        }
      ]
    },
    shortdef: ['to say', 'to tell', 'to tell, to say']
  },
  {
    meta: {
      id: 'tan:2',
      uuid: '6ad0961a-7c11-43ab-ad80-5e3a94c8e116',
      lang: 'es',
      sort: '2621448000',
      src: 'spanish',
      section: 'alpha',
      stems: ['tan', 'tan es así'],
      offensive: false
    },
    hom: 2,
    hwi: {
      hw: 'tan'
    },
    fl: 'pronoun',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                vrs: [
                  {
                    va: 'tan es así'
                  }
                ],
                dt: [['text', '{bc}so much so']]
              }
            ]
          ]
        ]
      }
    ],
    shortdef: ['so much so']
  },
  {
    meta: {
      id: 'tanto:4',
      uuid: 'f6712a94-2c57-4508-851f-89a9f6ab81d9',
      lang: 'es',
      sort: '2621463000',
      src: 'spanish',
      section: 'alpha',
      stems: [
        'tanto',
        'tanta',
        'en tanto',
        'entre tanto',
        'otro tanto',
        'por lo tanto',
        'tanto es así'
      ],
      offensive: false
    },
    hom: 4,
    hwi: {
      hw: 'tanto',
      prs: [
        {
          sound: {
            audio: 'tanto04sp'
          }
        }
      ]
    },
    ahws: [
      {
        hw: 'tanta',
        hwc: '-ta'
      }
    ],
    fl: 'pronoun',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                sn: '1',
                dt: [
                  ['text', '{bc}so much, so many '],
                  [
                    'vis',
                    [
                      {
                        t: 'tiene tanto que hacer',
                        tr: 'she has so much to do'
                      },
                      {
                        t: '¡no me des tantos!',
                        tr: "don't give me so many!"
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '2',
                vrs: [
                  {
                    va: 'en tanto',
                    vac: 'en ~'
                  }
                ],
                dt: [['text', '{bc}{a_link|while}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '3',
                vrs: [
                  {
                    va: 'entre tanto',
                    vac: 'entre ~'
                  }
                ],
                dt: [['text', '{bc}{a_link|meanwhile}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '4',
                vrs: [
                  {
                    va: 'otro tanto'
                  }
                ],
                dt: [
                  ['text', '{bc}again as much, again as many '],
                  [
                    'vis',
                    [
                      {
                        t: 'tiene un metro de ancho y otro tanto de altura',
                        tr: "it's a meter wide and a meter high"
                      },
                      {
                        t: 'otro tanto podría decirse de …',
                        tr: 'the same can be said of …'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '5',
                vrs: [
                  {
                    va: 'por lo tanto'
                  }
                ],
                dt: [['text', '{bc}{a_link|therefore}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '6',
                vrs: [
                  {
                    va: 'tanto es así'
                  }
                ],
                dt: [['text', '{bc}so much so']]
              }
            ]
          ]
        ]
      }
    ],
    shortdef: ['so much, so many', 'while', 'meanwhile']
  }
];

export default asi;
