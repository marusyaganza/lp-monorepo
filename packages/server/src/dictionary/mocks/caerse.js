onst caerse = [
  {
    meta: {
      id: 'caer',
      uuid: 'aac40ca2-64b3-4515-8a3d-91f11127a5b1',
      lang: 'es',
      sort: '2603583000',
      src: 'spanish',
      section: 'alpha',
      stems: [
        'caer',
        'caer en',
        'caer en desgracia',
        'caer enfermo',
        'caerle bien/mal a alguien',
        'caerle bien a alguien',
        'caerle mal a alguien',
        'caerse'
      ],
      offensive: false
    },
    hwi: {
      hw: 'caer',
      prs: [
        {
          sound: {
            audio: 'caer001sp'
          }
        }
      ]
    },
    fl: 'intransitive verb',
    def: [
      {
        sseq: [
          [
            [
              'sense',
              {
                sn: '1',
                dt: [
                  ['text', '{bc}to {a_link|fall} '],
                  [
                    'vis',
                    [
                      {
                        t: 'cay\u00f3 al suelo',
                        tr: 'he fell on the floor/ground'
                      },
                      {
                        t: 'lo dej\u00f3 caer',
                        tr: 'she dropped it'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '2',
                dt: [['text', '{bc}to drop away, to {a_link|slope}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '3',
                dt: [['text', '{bc}to {a_link|fall} (of night)']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '4',
                dt: [['text', '{bc}to {a_link|collapse}, to {a_link|fall}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '5',
                dt: [['text', '{bc}to {a_link|hang} (down)']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '6',
                dt: [
                  ['text', '{bc}to {a_link|realize}, to {a_link|understand} '],
                  [
                    'vis',
                    [
                      {
                        t: 'caer (en) que \u2026',
                        tr: 'to realize that \u2026'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '7',
                vrs: [
                  {
                    va: 'caer en',
                    vac: '~ en'
                  }
                ],
                dt: [
                  ['text', '{bc}to fall into (a trap, etc.) '],
                  [
                    'vis',
                    [
                      {
                        t: 'caer en el error de',
                        tr: 'to make the mistake of'
                      },
                      {
                        t: 'caer en manos de',
                        tr: 'to fall into the hands of'
                      },
                      {
                        t: 'caer en la tentaci\u00f3n',
                        tr: 'to give in to temptation'
                      }
                    ]
                  ]
                ]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '8',
                vrs: [
                  {
                    va: 'caer en desgracia'
                  }
                ],
                dt: [['text', '{bc}to {a_link|fall out of favor}']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '9',
                vrs: [
                  {
                    va: 'caer enfermo'
                  }
                ],
                dt: [['text', '{bc}to fall ill']]
              }
            ]
          ],
          [
            [
              'sense',
              {
                sn: '10',
                vrs: [
                  {
                    va: 'caerle bien/mal a alguien'
                  }
                ],
                sls: ['familiar'],
                dt: [
                  ['text', '{bc}to sit well/poorly with someone '],
                  [
                    'vis',
                    [
                      {
                        t: 'me caes bien',
                        tr: 'I like you'
                      }
                    ]
                  ]
                ]
              }
            ]
          ]
        ]
      }
    ],
    dros: [
      {
        drp: 'caerse',
        fl: 'reflexive verb',
        def: [
          {
            sseq: [
              [
                [
                  'sense',
                  {
                    dt: [
                      ['text', '{bc}to {a_link|fall} (down) '],
                      [
                        'vis',
                        [
                          {
                            t: 'se cay\u00f3 de rodillas',
                            tr: 'she fell to her knees'
                          }
                        ]
                      ]
                    ]
                  }
                ]
              ]
            ]
          }
        ]
      }
    ],
    suppl: {
      cjts: [
        {
          cjid: 'gppt',
          cjfs: ['cayendo', 'ca\u00eddo']
        },
        {
          cjid: 'pind',
          cjfs: ['caigo', 'caes', 'cae', 'caemos', 'ca\u00e9is', 'caen']
        },
        {
          cjid: 'pret',
          cjfs: [
            'ca\u00eda',
            'ca\u00edas',
            'ca\u00eda',
            'ca\u00edamos',
            'ca\u00edais',
            'ca\u00edan'
          ]
        },
        {
          cjid: 'pprf',
          cjfs: [
            'ca\u00ed',
            'ca\u00edste',
            'cay\u00f3',
            'ca\u00edmos',
            'ca\u00edsteis',
            'cayeron'
          ]
        },
        {
          cjid: 'futr',
          cjfs: [
            'caer\u00e9',
            'caer\u00e1s',
            'caer\u00e1',
            'caeremos',
            'caer\u00e9is',
            'caer\u00e1n'
          ]
        },
        {
          cjid: 'cond',
          cjfs: [
            'caer\u00eda',
            'caer\u00edas',
            'caer\u00eda',
            'caer\u00edamos',
            'caer\u00edais',
            'caer\u00edan'
          ]
        },
        {
          cjid: 'psub',
          cjfs: [
            'caiga',
            'caigas',
            'caiga',
            'caigamos',
            'caig\u00e1is',
            'caigan'
          ]
        },
        {
          cjid: 'pisb1',
          cjfs: [
            'cayera',
            'cayeras',
            'cayera',
            'cay\u00e9ramos',
            'cayerais',
            'cayeran'
          ]
        },
        {
          cjid: 'pisb2',
          cjfs: [
            'cayese',
            'cayeses',
            'cayese',
            'cay\u00e9semos',
            'cayeseis',
            'cayesen'
          ]
        },
        {
          cjid: 'fsub',
          cjfs: [
            'cayere',
            'cayeres',
            'cayere',
            'cay\u00e9remos',
            'cayereis',
            'cayeren'
          ]
        },
        {
          cjid: 'ppci',
          cjfs: [
            'he ca\u00eddo',
            'has ca\u00eddo',
            'ha ca\u00eddo',
            'hemos ca\u00eddo',
            'hab\u00e9is ca\u00eddo',
            'han ca\u00eddo'
          ]
        },
        {
          cjid: 'ppsi',
          cjfs: [
            'hab\u00eda ca\u00eddo',
            'hab\u00edas ca\u00eddo',
            'hab\u00eda ca\u00eddo',
            'hab\u00edamos ca\u00eddo',
            'hab\u00edais ca\u00eddo',
            'hab\u00edan ca\u00eddo'
          ]
        },
        {
          cjid: 'pant',
          cjfs: [
            'hube ca\u00eddo',
            'hubiste ca\u00eddo',
            'hubo ca\u00eddo',
            'hubimos ca\u00eddo',
            'hubisteis ca\u00eddo',
            'hubieron ca\u00eddo'
          ]
        },
        {
          cjid: 'fpin',
          cjfs: [
            'habr\u00e9 ca\u00eddo',
            'habr\u00e1s ca\u00eddo',
            'habr\u00e1 ca\u00eddo',
            'habremos ca\u00eddo',
            'habr\u00e9is ca\u00eddo',
            'habr\u00e1n ca\u00eddo'
          ]
        },
        {
          cjid: 'cpef',
          cjfs: [
            'habr\u00eda ca\u00eddo',
            'habr\u00edas ca\u00eddo',
            'habr\u00eda ca\u00eddo',
            'habr\u00edamos ca\u00eddo',
            'habr\u00edais ca\u00eddo',
            'habr\u00edan ca\u00eddo'
          ]
        },
        {
          cjid: 'ppfs',
          cjfs: [
            'haya ca\u00eddo',
            'hayas ca\u00eddo',
            'haya ca\u00eddo',
            'hayamos ca\u00eddo',
            'hay\u00e1is ca\u00eddo',
            'hayan ca\u00eddo'
          ]
        },
        {
          cjid: 'ppss1',
          cjfs: [
            'hubiera ca\u00eddo',
            'hubieras ca\u00eddo',
            'hubiera ca\u00eddo',
            'hubi\u00e9ramos ca\u00eddo',
            'hubierais ca\u00eddo',
            'hubieran ca\u00eddo'
          ]
        },
        {
          cjid: 'ppss2',
          cjfs: [
            'hubiese ca\u00eddo',
            'hubieses ca\u00eddo',
            'hubiese ca\u00eddo',
            'hubi\u00e9semos ca\u00eddo',
            'hubieseis ca\u00eddo',
            'hubiesen ca\u00eddo'
          ]
        },
        {
          cjid: 'fpsb',
          cjfs: [
            'hubiere ca\u00eddo',
            'hubieres ca\u00eddo',
            'hubiere ca\u00eddo',
            'hubi\u00e9remos ca\u00eddo',
            'hubiereis ca\u00eddo',
            'hubieren ca\u00eddo'
          ]
        },
        {
          cjid: 'impf',
          cjfs: ['-', 'cae', 'caiga', 'caigamos', 'caed', 'caigan']
        }
      ]
    },
    shortdef: ['to fall', 'to drop away, to slope', 'to fall (of night)']
  }
];

export default caerse;
