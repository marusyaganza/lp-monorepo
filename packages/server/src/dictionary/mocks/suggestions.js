const suggestions = [
  'ingest',
  'highest',
  'in jest',
  'inherit',
  'ginniest',
  'lithest',
  'Hengest',
  'nighest',
  'Genest',
  'richest',
  'behest',
  'inhere',
  'blithest',
  'inherent',
  'inherits',
  'archest',
  'gashest',
  'inhabit',
  'inhales',
  'inhered'
];

export default suggestions;
