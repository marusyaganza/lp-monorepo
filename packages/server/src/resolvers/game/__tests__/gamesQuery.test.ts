import {
  createTestServer,
  getDataFromGQL,
  getErrorMessageFromGQL
} from '../../../tests/helpers';
import { gameQueries } from '../../../tests/mocks/gqlQueries';
import { models } from '../../../tests/mocks/mockModels';

describe('game queries', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });
  afterAll(() => {
    jest.resetAllMocks();
  });

  test('games query', async () => {
    const { query } = createTestServer({ models });
    // @ts-expect-error: ts do not hanle this mocking case
    const res = await query({ query: gameQueries.gamesQuery });
    expect(getErrorMessageFromGQL(res)).toEqual(undefined);
    expect(getDataFromGQL(res)).toMatchSnapshot();
  });
});
