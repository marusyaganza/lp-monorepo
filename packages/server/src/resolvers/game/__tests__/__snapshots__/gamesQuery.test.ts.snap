// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`game queries games query 1`] = `
{
  "games": [
    {
      "desc": "Listen, Type, Master. Sharpen language skills through auditory engagement.",
      "id": "audio",
      "name": "WordEcho",
      "type": "AUDIO",
    },
    {
      "desc": "Define & Conquer.Match words with their meanings intuitively.",
      "id": "select_definition",
      "name": "LexiSense",
      "type": "SELECT_DEF",
    },
    {
      "desc": "Associate definitions with the right words. A captivating word puzzle adventure!",
      "id": "select_word",
      "name": "Definify: Match & Learn",
      "type": "SELECT_WORD",
    },
    {
      "desc": "Type words matching given definitions. Boost vocabulary in engaging rounds!",
      "id": "type_word",
      "name": "WordRecall: Define & Type",
      "type": "TYPE_WORD",
    },
    {
      "desc": "Identify objects from pictures and type their names to enhance vocabulary skills!",
      "id": "image",
      "name": "PictureWord Blitz",
      "type": "IMAGE",
    },
    {
      "desc": "Speak words from clues and images. Sharpen your vocabulary and speaking skills fast!",
      "id": "speaking",
      "name": "VoiceWord Challenge",
      "type": "SPEAKING",
    },
  ],
}
`;
